sql-plugin/src/main/scala/org/apache/spark/sql/catalyst/util/rapids/TimestampFormatter.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.util.rapids
+
+import java.text.ParseException
+import java.time.DateTimeException
+import java.time.format.DateTimeParseException
+
+// Copied from Spark: org/apache/spark/sql/catalyst/util/TimestampFormatter.scala
+// for for https://github.com/NVIDIA/spark-rapids/issues/6026
+// It can be removed when Spark3.3 is the least supported Spark version
+sealed trait TimestampFormatter extends Serializable {
```

Comment:
is this really needed or could we use spark version?    Maybe that woudl require shim so this is just easier?

Reply:
Oh, this file is not needed after refinement, removed.